src/dsp_tools/commands/ingest_xmlupload/bulk_ingest_client.py

```diff
@@ -59,29 +59,27 @@ def upload_file(
         filepath: Path,
     ) -> UploadFailure | None:
         """Uploads a file to the ingest server."""
-        try:
-            with open(self.imgdir / filepath, "rb") as binary_io:
-                content = binary_io.read()
-        except OSError as e:
-            err_msg = f"Cannot bulk-ingest {filepath}, because the file could not be opened/read: {e.strerror}"
-            logger.error(err_msg)
-            return UploadFailure(filepath, err_msg)
         url = self._build_url_for_bulk_ingest_ingest_route(filepath)
         headers = {"Content-Type": "application/octet-stream"}
-        timeout = 600
+        timeout = 1000
```

Comment:
*issue*: We have to make sure that this timeout is aligned with how our load balancers (LB) are currently configured

If both values are not equal we might end up in two different non-optimal or even problematic situations:
* timeout of LB is bigger: We loose time we could have used to upload bigger files. 
* timeout of LB is smaller: The request is killed by the LB and we have hard to understand error messages on both sides dsp-tools and dsp-ingest.

I would recommend a timeout value which is a couple of seconds smaller than the LB timeout.

Please double check with infra how far of we are from the optimal value.

Reply:
@SamuelBoerlin @LukasStoeckli What is the timeout of traefik? I ran `grep -ri "timeout"` in ops-deploy, but didn't find anything.

Reply:
@jnussbaum the [default is 60s](https://doc.traefik.io/traefik/routing/entrypoints/#transport), which is what we're currently using for all publicly exposed endpoints.

Reply:
Thanks for your input. Resolved with https://github.com/dasch-swiss/dsp-tools/commit/d3d15d7ba90a62799c182bbedd2116eadc3d84fa